scripts/run-swig

```diff
@@ -2,12 +2,13 @@
 set -e
 ROOTDIR=$(cd $(dirname $(dirname $0)) && pwd -P)
 
-swig -c++ -package org.openobservatory.measurement_kit                         \
+swig -c++ -package org.openobservatory.measurement_kit.swig                    \
         -java jni/ooni_test_wrapper.i
```

Comment:
@lorenzoPrimi, also note that I changed the package of classes generated directly by Swig. The classes that netprobe-android is currently using are those generated by Swig, thus if you want to keep using them as opposed to using the more usable wrappers in the `.nettests` package, know that you need to change the package.